project/cmake/scripts/common/addon-helpers.cmake

```diff
@@ -34,6 +39,17 @@ macro (build_addon target prefix libs)
     SET_TARGET_PROPERTIES(${target} PROPERTIES PREFIX "lib")
   ENDIF(OS STREQUAL "android")
 
+  GET_TARGET_PROPERTY(LIBRARY_LOCATION ${target} LOCATION)
```

Comment:
this has been deprecated by http://www.cmake.org/cmake/help/v3.0/policy/CMP0026.html
see https://github.com/wsnipex/xbmc/commit/4284ce9c4675dd52354c7e1e5f50c66fe9a0a4df for a POC that works around it on linux. Something similar would be needed for the VS project file.